package.json

```diff
@@ -30,36 +31,39 @@
   },
   "homepage": "https://github.com/babel/babel-library-boilerplate",
   "devDependencies": {
-    "babel": "^5.0.0",
-    "babelify": "^6.0.0",
-    "browserify": "^8.1.1",
-    "chai": "^2.0.0",
+    "babel": "^5.2.17",
```

Comment:
> But as I stated earlier I get trouble with changing version numbers on all of the dependencies because of nodejJS 12.2. It's require a lot of different versions. At least in my end. So I skipped that part.

Ah, makes sense. I really shouldn't have assumed that you weren't familiar with the `^` operator – that's my bad! Definitely make whatever changes are necessary to get the build to pass! :)